src/vorta/views/partials/treemodel.py

```diff
@@ -994,3 +995,32 @@ def lessThan(self, left: QModelIndex, right: QModelIndex) -> bool:
         data1 = self.choose_data(left)
         data2 = self.choose_data(right)
         return data1 < data2
+
+    def setFilterFixedString(self, pattern: str):
```

Comment:
Does overriding this existing method provide a benefit? Otherwise I would choose a more fitting name since we will filte with an elaborate syntax eventually.